topics/software-challenge/index.md

```diff
@@ -0,0 +1,15 @@
+---
+aliases: sc, softwarechallenge
+display_name: Software Challenge
+github_url: https://github.com/software-challenge
+logo: software-challenge.png
+released: 2004
+short_description: Programming competition for students, held by the CAU Kiel and FH Wedel.
+topic: software-challenge
```

Comment:
I think there are a couple of conflicts that could arise with a generic name like this. If we change the main topic to `software-challenge-de` that would solve them for me. Also, the alias `sc` is far too broad to apply here.

Note: with a topic name change, you will have to update the directory name as well.

```suggestion
aliases: softwarechallengede
display_name: Software Challenge
github_url: https://github.com/software-challenge
logo: software-challenge.png
released: 2004
short_description: Programming competition for students, held by the CAU Kiel and FH Wedel.
topic: software-challenge-de
```